lib.ftd

```diff
@@ -3625,3 +4025,338 @@ align-self: start
+role: $fonts.size-23
+color: $inherited.colors.text
+margin-bottom.px: 10
+text-align: center
+width: fill-container
+
+-- ftd.column:
+width.fixed.px: 300
+background.image: $bg-image
+height.fixed.px: 300
+align-self: center
+
+-- ftd.row:
+align-content: center
+width: fill-container
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.top-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.custom.one
+padding-vertical.px: 13
+padding-horizontal.px: 17
+
+-- end: ftd.row
+
+-- ftd.row:
+align-content: center
+width: fill-container
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.bottom-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.background.step-1
+padding-vertical.px: 13
+padding-horizontal.px: 17
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- ftd.image:
+src: $works-cell-desktop.right-src
+padding-right.px: 12
+align-self: center
+width: fill-container
+margin-right.px: -44
+
+-- end: ftd.row
+
+-- end: works-cell-desktop
+
+
+
+
+
+
+
+
+
+
+-- component works-cell-mobile:
+caption title:
+integer number:
+ftd.image-src left-src:
+ftd.image-src right-src:
+optional ftd.image-src left-mob-src:
+optional ftd.image-src right-mob-src:
+string top-cell:
+string bottom-cell:
+optional integer index:
+optional integer reset:
+
+-- ftd.column:
+width: fill-container
+margin-top.px: 25
+margin-bottom.px: 24
+
+-- ftd.row:
+width: fill-container
+spacing: space-between
+
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src == NULL }
+src: $works-cell-mobile.left-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src != NULL }
+src: $works-cell-mobile.left-mob-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src == NULL }
+src: $works-cell-mobile.right-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src != NULL }
+src: $works-cell-mobile.right-mob-src
+width.fixed.percent: 27
+height: auto
+
+-- end: ftd.row
+
+-- ftd.row:
+background.solid: $inherited.colors.border
+padding-vertical.px: 7
+padding-horizontal.px: 12
+width: fill-container
+spacing.fixed.px: 6
+
+-- ftd.row:
+spacing.fixed.px: 2
+
+-- ftd.integer: $works-cell-mobile.number
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+
+-- ftd.text: .
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- ftd.text: $works-cell-mobile.title
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+width: fill-container
+
+-- end: ftd.row
+
+-- ftd.column:
+width: fill-container
+padding-vertical.px: 17
+padding-horizontal.px: 10
+
+-- ftd.text: $works-cell-mobile.top-cell
+role: $inherited.types.blockquote
+color: $inherited.colors.text-strong
+text-align: center
+width: fill-container
+
+-- ftd.row:
+width.fixed.percent: 70
+align-self: center
+height.fixed.px: 1
+background.solid: $inherited.colors.border
```

Comment:
Why is `border` color used for `background`?

Reply:
because we need a color value that color code is matched with border color

Reply:
@Arpita-Jaiswal resolving this - as this color scheme needed some more colors where some colors added in different color types.